homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -123,6 +124,9 @@
     15: STATE_RETURNING,
     16: STATE_CLEANING,
     17: STATE_CLEANING,
+    18: STATE_CLEANING,
+    100: STATE_CHARGING_COMPLETE,
+    101: STATE_OFFLINE,
```

Comment:
Actually, these two latest should not be new states as this dict is, to my understanding, used to map to states to more generic ones to allow interoperability with other integrations (e.g., google home or alexa). Maybe someone else can chime in and say the final word on this?